### DISCUSSION TITLE
Are there simple examples of predictions of outcome in future months, exemplified by the Sunspots dataset？

### DISCUSSION THREAD

**Discussion by jacklau1982** [timeseriesAI/tsai#557] [Q&A]
Are there simple examples of predictions of outcome in future months, exemplified by the Sunspots dataset？

**Comment by oguiza** [comment]
Hi @jacklau1982 ,
I'm working on a tutorial nb that will demonstrate how to use `tsai` for single and multi-step forecasts using the Sunspots dataset.
